lib/services/devices/registrationUtils.js

```diff
@@ -136,7 +136,6 @@ function sendUnregistrationsNgsi2(deviceData, callback) {
     const options = {
         url: cbHost + '/v2/registrations/' + deviceData.registrationId,
         method: 'DELETE',
-        json: true,
```

Comment:
This was already deleted by https://github.com/telefonicaid/iotagent-node-lib/pull/1178

Reply:
This PR is a PoC. Changes in #1178 were included here in order to check if changes on tests were capturing the problem exposed and, after applying the patch, if it continues or not.